backend/src/test/java/com/redhat/cloud/notifications/TestLifecycleManager.java

```diff
@@ -40,7 +40,7 @@ public void stop() {
     }
 
     void setupPostgres(Map<String, String> props) throws SQLException {
-        postgreSQLContainer = new PostgreSQLContainer<>("postgres");
+        postgreSQLContainer = new PostgreSQLContainer<>("postgres:12");
```

Comment:
Postgres 12 is not currently used anywhere (ephemeral, stage or prod). Should we align the version with prod?

Reply:
Without a qualifier it just takes anything. So setting to something we use in prod may be good. But only the major, otherwise it causes too much churn (IMO)

Reply:
Specifying the major is definitely better than using whatever latest is 👍Here's the major currently used on prod:
```suggestion
        postgreSQLContainer = new PostgreSQLContainer<>("postgres:11");
```